gensim/test/test_corpora_dictionary.py

```diff
@@ -179,6 +177,58 @@ def test_doc2bow(self):
         # unicode must be converted to utf8
         self.assertEqual(d.doc2bow([u'\u017elu\u0165ou\u010dk\xfd']), [(0, 1)])
 
+    def test_saveAsText(self):
+        """`Dictionary` can be saved as textfile. """
+        tmpf = get_tmpfile('save_dict_test.txt')
+        small_text = [["prvé", "slovo"],
+                      ["slovo", "druhé"],
+                      ["druhé", "slovo"]]
+
+        d = Dictionary(small_text)
+
+        d.save_as_text(tmpf)
+        with open(tmpf) as file:
+            serialized_lines = file.readlines()
+            self.assertEqual(serialized_lines[0], "3\n")
+            self.assertEqual(len(serialized_lines), 4)
+            # We do not know, which word will have which index
+            self.assertEqual(serialized_lines[1][1:], "\tdruhé\t2\n")
+            self.assertEqual(serialized_lines[2][1:], "\tprvé\t1\n")
+            self.assertEqual(serialized_lines[3][1:], "\tslovo\t3\n")
+
+        d.save_as_text(tmpf, sort_by_word=False)
+        with open(tmpf) as file:
+            serialized_lines = file.readlines()
+            self.assertEqual(serialized_lines[0], "3\n")
+            self.assertEqual(len(serialized_lines), 4)
+            self.assertEqual(serialized_lines[1][1:], "\tslovo\t3\n")
+            self.assertEqual(serialized_lines[2][1:], "\tdruhé\t2\n")
+            self.assertEqual(serialized_lines[3][1:], "\tprvé\t1\n")
+
+    def test_loadFromText(self):
```

Comment:
The first part is for format, the second is for new. I create two different functions for better verbosity.